definitions/tools/mark_duplicates_and_sort.cwl

```diff
@@ -37,6 +37,10 @@ inputs:
         type: File
         inputBinding:
             position: 1
+    markdup_assume_sort_order:
+        type: string
+        inputBinding:
+            position: 5
```

Comment:
I'd favor giving this a default value of "queryname", which is what it was previously (and what several pipelines expect).  Then you can explicitly override it in the RNAseq pipeline, which now inputs "coordinate"